src/WordPress/Blueprints/Runtime/NativePHPRuntime.php

```diff
@@ -35,13 +35,57 @@ public function resolvePath( string $path ): string {
 		if ( ! strlen( $path ) ) {
 			return $this->getDocumentRoot();
 		}
-		if ( $path[0] === '/' ) {
+		if ( $this->isAbsolutePath($path) ) {
 			return $path;
 		}
 
 		return join_paths( $this->getDocumentRoot(), $path );
 	}
 
+    private function isAbsolutePath(string $path): bool {
```

Comment:
Since it's ported from WordPress, let's add an attribution in the method comment.

Reply:
This could use its own set of tests. Let's make it public for now and add a few tests to confirm it indeed works as expected. Later on we'll remove it to some Filesystem-related class.

Oh! Or it could be a function and not a method, in which case you'd add it to the `functions.php` file.

Reply:
On this note - actually there is a method in the FileSystem API that does the same thing. However, I have to review if their inner workings are intertidal.

See: Filesystem::isAbsolutePath

Reply:
Well, the FileSystem method seems less robust. But maybe it is sufficient? LMK which do you prefer @adamziel and I'll adjust accordingly.

Reply:
> Also, please review if the method Filesystem\Path::makeAbsolute could be a viable alternative to most of the logic in the resolvePath method. I'll push a possible change, for you to review.

Nice, you're right!